addons/interaction/CfgVehicles.hpp

```diff
@@ -35,6 +35,32 @@ class CfgVehicles {
                 icon = "\a3\ui_f\data\IGUI\Cfg\Actions\eject_ca.paa";
                 selection = "pelvis";
 
+                class ACE_PassMagazine {
+                    displayName = CSTRING(PassMagazine);
+                    condition = QUOTE([ARR_3(_player,_target,primaryWeapon _target)] call FUNC(canPassMagazine) || [ARR_3(_player,_target,handgunWeapon _target)] call FUNC(canPassMagazine));
```

Comment:
Parent condition can be empty string or just `true`.  
If there are no children, it won't show because statement is `{}`
Will save checking the conditions twice.